docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -40,17 +42,29 @@ ALTER TABLE `geneset` AUTO_INCREMENT = 1;
 4- Restart cBioPortal (restart webserver) to clean-up any cached gene lists.
 
 5- To import gene data type the following commands when in the folder `<cbioportal_source_folder>/core/src/main/scripts`:
+### Human genes
+```
+export PORTAL_HOME=<cbioportal_configuration_folder>
+export JAVA_HOME=<jre_installation_folder>
+./importGenes.pl --hgnc <gene_info.txt> --genome-build <GRCh37>
+```
+### Mouse genes
 ```
 export PORTAL_HOME=<cbioportal_configuration_folder>
 export JAVA_HOME=<jre_installation_folder>
-./importGenes.pl --genes <ncbi_species.gene_info> --gtf <gencode.v25.annotation.gtf> --genome-build <GRCh37>
+./importGenes.pl --genes <Mus_musculus.gene_info.gz> --gtf <gencode.v25.annotation.gtf> --genome-build <GRCh37>
 ```
 **IMPORTANT NOTE**: 
 1. The **reference_genome** table needs to be populated before updating the **gene** table. Further details can be found in [this document](import-reference-genome.md). 
 2. Use **--species** option when importing genes for a species other than human
 3. Use the **gene** table if you query information such as hugo symbols, types of the gene 
 4. Use **reference_genome_gene** table if you query information such as chromosome, cytoband, exonic length, or the start or end of the gene
 5. Load genes only to the **reference_genome_gene** table without updating the **gene** table, please use the following command:
+### Human genes
+```
+./importGenes.pl --gtf <gencode.v27.annotation.gtf> --genome-build <GRCh38>
+```
+### Mouse genes
```

Comment:
this is wrong @yichaoS ... "GRCh38" is a human assembly, not mouse

Also ... is it correct to be using the same gencode gtf file both for human and for mouse?

Reply:
@sheridancbio we didn't update anything related to mouse genes....But was that a mistake we always have then? Not familiar when mouse genes. And for the gtf file not used for human genes anymore (@mandawilson ?) since we don't generate gene length in the new table? Should we just delete that parameter?

Reply:
@sheridancbio agreed. I created this issue here: https://github.com/cBioPortal/cbioportal/issues/8678 and tagged you me and @mandawilson for future reference.